api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3053,6 +3053,48 @@ message Unknown {
   string Data = 4 [(gogoproto.jsontag) = "data"];
 }
 
+// OSType is the same as teleport.devicetrust.v1.OSType.
+// Duplicated because gogo doesn't play well with protoc-gen-go.
+enum OSType {
+  OS_TYPE_UNSPECIFIED = 0;
+  // Linux.
+  OS_TYPE_LINUX = 1;
+  // macOS.
+  OS_TYPE_MACOS = 2;
+  // Windows.
+  OS_TYPE_WINDOWS = 3;
+}
+
+// DeviceTrustMetadata groups device information for events.
+message DeviceTrustMetadata {
+  // ID of the device.
+  string device_id = 1;
+  // OS of the device.
+  OSType os_type = 2;
+  // Device inventory identifier.
+  string asset_tag = 3;
+  // Device credential identifier.
+  string credential_id = 4;
+}
+
+// DeviceTrust is a device-related event.
+// The event type (Metadata.Type) for device events is always "device". See the
+// event code (Metadata.Code) for its meaning.
+message DeviceTrust {
```

Comment:
Could you rebase into an earlier master? The files you changed don't exist anymore, so there's a chance the import cycle is done (or at least diminished).